osd_cluster_admin/osd-cluster-autoscaling.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="osd-cluster-autoscaling"]
+= Cluster autoscaling
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: osd-cluster-autoscaling
+
+toc::[]
+
+Applying autoscaling to an {product-title} cluster involves configuring a cluster autoscaler and then configuring machine autoscalers for each machine type in your cluster.
```

Comment:
should it be `machine pool` instead of `machine type`? (same for the rosa cluster)

Reply:
CC: @osherdp for this comment.